### DISCUSSION TITLE
Field-Collection in gridview

### DISCUSSION THREAD

**Discussion by jarinirv** [pimcore/pimcore#14862] [Q&A / Support]
I've class 'Product', Panel Environment and under that Field-Collection 'Materials'.  Field collection as different products may contain different materials and you can only add allowed types defined in class 'Material'. You also have to give weight of the material you've added and then there's calc that tells what is the percentage of products overall weight. Let's say I've product that contains steel, paint and plastic and I've added those to 'Materials' along with their weights and got percentages.  When I add that 'Materials' in Grid Options it already says in preview Not supported. Is there a way to show that field collections content in gridview?

**Comment by jarinirv** [comment]
Answering to myself. It has become into my knowledge from other source that this is not possible.

**Reply by jeykodev** [reply]
@jarinirv do you have more context regarding the reason, why this should not be possible?

**Reply by jarinirv** [reply]
@jeykodev Not sure if I understand what you mean and this being almost year old case I faintly recall that it was told to be impossible by our integrator that went out of business late last year.
